### PR TITLE
Parse includes list when compiler language is not Englich

### LINKED ISSUE #65: parseIncludesList regex only working for english locale
the english compiler output is "Note: including file", though the german language output is "Hinweis: Einlesen der Datei" ... not idea about non-european languages ...

cotire seems to have some code, which might help:
https://github.com/facebook/hhvm/blob/master/CMake/cotire.cmake#L898

### PR DESCRIPTION
**Pull request by webmaster128**
This adds a test generated by a German cl output.

Closes #65.

This is WIP and should not be merged until all lights are green in CI.

### CODE REVIEW

**Comment by webmaster128** [comment]
Okay, I guess this is done :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ clcache changelog
 ## Upcoming release
 
  * Bugfix: Supportuse of py2exe with Python 3.4
+ * Bugfix: Support includes parsing when cl language is not Englich (GH #65)
```
Comment (frerich) [inline CHANGELOG.md]: Typo here, which reminds me of the germanischen Version of Life of Brian.

Comment (frerich) [inline CHANGELOG.md]: Germanischen... That is what I get for trying to review code from a cell phone.

Comment (webmaster128) [inline CHANGELOG.md]: Right, thanks. Could you merge #66 first? Then we don't get merge conflicts in this part.

**Comment by webmaster128** [comment]
Rebased onto current master. Fixed "English" -> "English" in commit message and changelog line. Anything else?

**Comment by frerich** [comment]
Looks good to me, thanks!

**Pull request merged**